### PR TITLE
detector return heatmaps

### PR DESCRIPTION
**Pull request by jin-s13**

### CODE REVIEW

File: mmpose/datasets/datasets/bottom_up/bottom_up_coco.py [hunk 1]
```diff
@@ -204,7 +204,7 @@ def evaluate(self, outputs, res_folder, metric='mAP', **kwargs):
             num_keypoints: K
 
         Args:
-            outputs (list(preds, scores, image_path)):Output results.
+            outputs (list(preds, scores, image_path, output_cpu)):
```
Comment (innerlee) [inline mmpose/datasets/datasets/bottom_up/bottom_up_coco.py:207]: could `output_cpu` be more specific/concrete? say, things like `heatmap` etc.

File: mmpose/models/detectors/bottom_up.py [hunk 2]
```diff
@@ -243,7 +243,9 @@ def forward_test(self, img, img_metas, **kwargs):
         image_path = []
         image_path.extend(img_metas['image_file'])
 
-        return results, scores, image_path
+        output_cpu = aggregated_heatmaps.clone().cpu().numpy()
```
Comment (innerlee) [inline mmpose/models/detectors/bottom_up.py:246]: should clone -> detach?

**Comment by innerlee** [comment]
the file `tests/data/smpl/test_joint_regressor.npy` is too large. try to use a smaller test data size

**Comment by jin-s13** [comment]
> the file `tests/data/smpl/test_joint_regressor.npy` is too large. try to use a smaller test data size

maybe not related to this PR

File: mmpose/datasets/datasets/bottom_up/bottom_up_coco.py (hunk 1 above)
Comment (jin-s13) [reply mmpose/datasets/datasets/bottom_up/bottom_up_coco.py]: changed to `output_heatmap`

File: mmpose/models/detectors/bottom_up.py (hunk 2 above)
Comment (jin-s13) [reply mmpose/models/detectors/bottom_up.py]: Yes. should be `detach()`

**Comment by jin-s13** [comment]
> the file `tests/data/smpl/test_joint_regressor.npy` is too large. try to use a smaller test data size

Yes. This file should not be uploaded. I will delete it.

**Review by innerlee — APPROVED** [review]

**Pull request merged**